evaluator/src/main/kotlin/PackageRule.kt

```diff
@@ -133,22 +133,23 @@ open class PackageRule(
         }
     }
 
-    fun issue(severity: Severity, message: String) = issue(severity, pkg.id, null, null, message)
+    fun issue(severity: Severity, message: String, howToFix: String) =
```

Comment:
PR: Why not delay the commit / PR until the client code is crafted, because that would provide a proof that the API makes sense as is.

Reply:
I have added rendering in the static HTML report to the PR.

Reply:
As mentioned above my concern wasn't really about the change in the violation model but rather in the change of those issue functions.

Reply:
But if `howToFix` is stored next to `severity` and `message` in the violation class, what makes it so special, that it cannot be set in the same way? In the end it's the same as message, but with different content.

Reply:
I heard someone talking about package manager specific hints. 
I assumed that more complex logic would be needed to create a hint compared to creating a message. Thus the hint creation would be different in terms of readability.

I just feared that looking at a rule the code would be made up 95% for the hint and 5% for the rule. Did you consider that?

Anyhow, I'm ok with this approach as well as it is easily transformable into the other when it turns out to make sense.

Reply:
In this case it's up to the user to organize the code in a good way if message creation gets complex. If you are ok with the change, can you approve it?